homeassistant/components/owlet/entity.py

```diff
@@ -0,0 +1,20 @@
+"""Base class for Owlet entities."""
+
+from homeassistant.helpers.entity import Entity
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .coordinator import OwletCoordinator
+
+
+class OwletBaseEntity(CoordinatorEntity[OwletCoordinator], Entity):
```

Comment:
```suggestion
class OwletBaseEntity(CoordinatorEntity[OwletCoordinator]):
```
`CoordinatorEntity` already inherits from `Entity`.